### DISCUSSION TITLE
I am glad this exists, is this project somewhat safe to rely on? I am willing to help or donate but I don't want it to disappear like the old rport.

### DISCUSSION THREAD

**Discussion by grahamIT** [openrport/openrport#38] [General]
I am willing to help or donate but I don't want it to disappear like the old rport. Additionally I have a few suggestions:
 - rename the project entirely and rebrand. You really don't want to risk getting sued.
 - Create a roadmap and have tasks that contributors can work on
 - Create a donation page with current financial status. Don't be afraid to prompt for donations or offer a version that can be purchased to support the project.

Overall, this looks pretty good. I think its way better security wise than something like TacticalRMM.

**Comment by alhdo** [comment]
Hi @grahamIT Thank you for your advice.
I will be working on that and your help is appreciated. The roadmap will be added soon and the donation page. I am dedicated to keep the project opensource and onboard new maintainer so it never disappear.

**Comment by sevenwalls2081** [comment]
Count me in as someone who would like to donate or even to @grahamIT point, pay for a version to help support.  I am no coder but I appreciate the work you all have put into this.  I have been searching for an RMM-like solution since the original project went closed source.  So happy to see someone keeping it alive.  This home-labber is greatly appreciative.

**Comment by alhdo** [comment]
Thanks for your support @sevenwalls2081. I set up a donation page from github. How much would you consider a good amount that works for you for a donation? I am trying to put them down with some benefits i can give back to the supporter.

**Comment by d3al** [comment]
I'm glad to see this project became OSS. I was very disappointed by the rug pull realVNC did to the point I wouldn't purchase anything from them going forward.

What I like about this project is how simple it was and had no interest in the more advanced features, all I care about is the ability to create tunnels with SSH/RDP/VNC, service forwarding, API and execute remote commands. I'm still on 9.12 and just came across this project today so it's good to see there is some activity and I really hope this project stays alive and maintained. There is a lot of UI polish that needs done so I hope that is being worked on, there could be some quality of life improvements but overall this IMO is a very good piece of technology.

I assume I can upgrade to 9.14 from 9.12 without issue? 9.12 clients connect to 9.14 server?

**Comment by alhdo** [comment]
Hello @d3al thank you for ou message. Yes i am maintaining the project most of my time right now is being spent on having an OSS front-end. i hope to get out a first version soon. For your question yes 9.12 does connect on 9.14.

**Comment by SinisterSpatula** [comment]
So glad to see an openrport!  I tried out the original project and was very disappointed when they removed it.  I thought it was a super cool technology and there's nothing else like it.  Thank you very much for all the efforts on this!

**Comment by IOTech17** [comment]
@alhdo 

Any update on your side? How is the development going on?

**Comment by alhdo** [comment]
Hello @IOTech17  i have been collaborating with @joancabezas on the front of the project. I had some issue to figure out for myself now i got everything in order, the development will go faster. We are also looking for help to go even further. We will release the dev version of it to the public to get more member of the community involved. 

Thanks

**Reply by IOTech17** [reply]
Thank you for the update, this is very food to know.

I can help on the docker side of things, I used to dockerized rport in ubuntu then in alpine to make it work with guacd (basically running two services in docker using supervisord).
Let me know once the new UI is avalable and I will dockerized it and test it

**Comment by bmcgonag** [comment]
I'm thrilled to see this. I am ready to donate monetarily, if you can setup a donation page, paypal, or whatever. I'd love to start a monthly donation to make sure this is kept open source and continues moving forward.  Thank you so much for all the work on this!

**Reply by alhdo** [reply]
Hello @bmcgonag thank for your support. We are working on the new front-end also. If you are web dev feel free to contribute. The link to the [donation](https://github.com/sponsors/openrport) is on the organization page. I ll put it there https://github.com/sponsors/openrport . We are commited to keep this project alive and adding new features based on the community. 
Thanks

**Reply by bmcgonag** [reply]
@alhdo Just donated! Thanks for sharing the link. I looked on the README, maybe I missed it, but would be worth adding it there as well.

**Reply by alhdo** [reply]
Great big thanks. I will add it on the readme. Thanks for the tips.

**Comment by mauriva** [comment]
Looking at the current github page, the project seems to have been at a standstill for some time.
Will the frontend mentioned be published here or on a separate project? I couldn't find any link...

**Reply by alhdo** [reply]
Hello, the ui is work in progress and already published to the organization page.

**Reply by alhdo** [reply]
https://github.com/openrport/openrport-ui

**Reply by mauriva** [reply]
Thank you!

**Comment by d3al** [comment]
Thanks, I didn't realize there was a separate repo just for the UI.

I'm running 9.14 in production and wanted to avoid unnecessary changes
to rport and I wanted to know if the new UI is ready to swap out? If
not do you have an estimate when it should be ready enough for day to
day?

Also I would throw some screen shots on the git page so we have an idea
what to expect.

On Wed, 2024-10-16 at 08:34 -0700, mauriva wrote:
> Thank you!
> —
> Reply to this email directly, view it on GitHub [1], or unsubscribe
> [2].
> You are receiving this because you were mentioned.Message ID:
> <openrport/openrport/repo-
> ***@***.***>


[1] view it on GitHub
    https://github.com/openrport/openrport/discussions/38#discussioncomment-10961548
[2] unsubscribe

**Reply by alhdo** [reply]
Hello, currently i am working alone on it right now. Alongside my day to day job so i gave it as much as possible time. It is not ready to swap out yet. I am trying to get all the more used functionalities to be ready first. 
Thanks i will add some screenshot in the readme but basically it is the same design as the old one. I tried to keep it as same as possible so the swap would be seamless. 

If you are a front-end dev. You are welcome to collaborate. Don't hesitate to join the discord to get updated more frequently.
